tests/cases/fourslash/moveToNewFile.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: /a.ts
+////import { a, b } from "./other";
```

Comment:
[I'm sure this is clear from the product code, but I haven't read it in detail.]  What happens if you extract an import statement.  Is that disallowed?

Reply:
What about a return statement?  Can a return statement be moved to another file?

Reply:
> What happens if you extract an import statement

I think we should allow this, though there's a bug if the moved import is still needed in the old file. #23968

> Can a return statement be moved to another file?

Only top-level statements can be moved, and 'return' at top-level is an errorany way.